docs/language/learn-ql/cpp/dataflow.rst

```diff
@@ -244,6 +244,49 @@ The following data flow configuration tracks data flow from environment variable
    select fopen, "This 'fopen' uses data from $@.",
      getenv, "call to 'getenv'"
 
+The following taint tracking configuration tracks data from a call to ``ntohl`` to an array index operation. It uses the ``Guards`` library to recognize expressions that have been bounds checked and avoid propagating taint through them. It also uses ``isAdditionalTaintStep`` to add flow from loop bounds to loop indexes.
```

Comment:
Second sentence doesn't quite flow for me. Does this make sense?

It uses the ``Guards`` library to recognize expressions that have been bounds checked, and therefore don't propagate taint.

Reply:
That reads differently to me, since it implies that taint not propagating through bounds-checked expressions when using this confiugration is a property of the expression rather than a property of the taint-tracking configuration. Would this be better:

It uses the `Guards` library to recognize expressions that have been bounds-checked and prevent taint from propagating through them.

Reply:
Yeah that's better. I misunderstood it before, but it makes more sense now. Thanks.